src/main/java/com/rakovets/course/java/core/practice/concurrency/threads/Consumer.java

```diff
@@ -0,0 +1,41 @@
+package com.rakovets.course.java.core.practice.concurrency.threads;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalTime;
+import java.util.Queue;
+
+public class Consumer implements Runnable {
+    Path path = Paths.get("src","main","resources","practice","text");
+    boolean isAlive = true;
+
+    public void run(){
+        while (isAlive = true)
+        try {
+            System.out.println("started");
+            FileWriter writer = new FileWriter(String.valueOf(path),true);
+            if (Store.list.size() > 0){
+                int priority = Store.list.element();
+                if (priority == -1){
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"